crates/js-sys/src/lib.rs

```diff
@@ -3641,6 +3641,31 @@ impl JsString {
     ) -> impl ExactSizeIterator<Item = u16> + DoubleEndedIterator<Item = u16> + 'a {
         (0..self.length()).map(move |i| self.char_code_at(i) as u16)
     }
+
+    /// If this string consists of a single UTF-8 codepoint, then this function
+    /// returns a Rust representation of that character without any allocations.
```

Comment:
I think it's a bit awkward to call this a "Rust representation". Perhaps it's better to say something like "converts it into a Rust `char` (without doing any allocations)."

Reply:
Fixed.